samples/snippets/update_lite_topic_example.py

```diff
@@ -35,39 +40,44 @@ def update_lite_topic(project_number, cloud_region, zone_id, topic_id):
     # cloud_region = "us-central1"
     # zone_id = "a"
     # topic_id = "your-topic-id"
+    # reservation_id = "your-reservation-id"
 
     cloud_region = CloudRegion(cloud_region)
     location = CloudZone(cloud_region, zone_id)
     topic_path = TopicPath(project_number, location, topic_id)
```

Comment:
those all need to be changed so that location can be either zonal or regional

Reply:
Updated.